scc/provider/resource_subaccount.go

```diff
@@ -78,6 +77,18 @@ __Further documentation:__
 				Computed:            true,
 				Optional:            true,
 			},
+			"connected": schema.BoolAttribute{
+				MarkdownDescription: `Indicates whether the subaccount is connected to the Cloud Connector.
+This value depends on the **tunnel state**:
+- If the tunnel is in state *Connected*, this will be *true*.
+- If the tunnel is in state *Disconnected*, this will be *false*.
+
+**Note**:
+"In case of a *ConnectFailure*, you may attempt to recover the connection by first setting connected to false 
+and then back to true, which will retry connecting the subaccount.`,
+				Computed: true,
```

Comment:
I think computed won't work, The parameter must be optional and have a default value that is true. This enables the user to create a subaccount in the state disconnected right from the start. With the current implementation, the user first creates the subaccount and then needs to update it in case that it should be disconnected.

This would also make more sense in the case of a connection failure: The user configured the value to true, but the state in the tunnel signals an error. Then the first change needed is to set the value to false to retrive a defined state namely disconnected. After that (and probably after validating that verything should work, the user would switch it back to true. 

Do you agree or am I missing something?